src/openafval/trash/admin.py

```diff
@@ -0,0 +1,118 @@
+from django.contrib import admin
+from django.http import HttpRequest
+from django.utils.translation import gettext_lazy as _
+
+from .models import (
+    BagObject,
+    Container,
+    ContainerType,
+    Emptying,
+    Entity,
+    EntityObjectManagement,
+)
+
+
+class ReadOnlyMixin:
+    self: admin.ModelAdmin
+
+    def has_change_permission(self, request: HttpRequest, obj=None):
+        return False
+
+    def has_add_permission(self, request: HttpRequest, obj=None):
+        return False
+
+    def has_delete_permission(self, request: HttpRequest, obj=None):
+        return False
+
+
+@admin.register(BagObject)
+class BagObjectAdmin(ReadOnlyMixin, admin.ModelAdmin):
+    list_display = (
+        "identifier",
+        "address",
+    )
+
+
+@admin.register(Entity)
+class EntityAdmin(ReadOnlyMixin, admin.ModelAdmin):
+    list_display = ("identifier", "bsn", "name")
+
+
+@admin.register(EntityObjectManagement)
+class EntityObjectManagementAdmin(ReadOnlyMixin, admin.ModelAdmin):
+    list_display = (
+        "identifier",
+        "entity",
+        "bag_object",
+        "start_date",
+        "end_date",
+    )
+
+
+@admin.register(ContainerType)
+class ContainerTypeAdmin(ReadOnlyMixin, admin.ModelAdmin):
+    list_display = (
+        "type",
+        "description",
+        "fraction_identifier",
+        "fraction_description",
+    )
+
+    def fraction_identifier(self, obj):
+        return obj.fraction.identifier
+
+    def fraction_description(self, obj):
+        return obj.fraction.description
+
+
+@admin.register(Container)
+class ContainerAdmin(ReadOnlyMixin, admin.ModelAdmin):
+    list_display = (
```

Comment:
- Consider adding something like "fraction_type" to `list_display` and define:
    ```
    @admin.display(description=_("fraction"))
    def fraction_type(self, obj):
        return obj.type.fraction.description if obj.type and obj.type.fraction else "-"
    ```

Could be useful to see this in the admin